outfit.js

```diff
@@ -20,16 +20,3 @@ class Outfit {
     this.background = background
   }
 };
-
-
-// function valueCompare() {
-//  if (parseInt(maxInput.value) < parseInt(minInput.value) &&
-//     (minInput.value != "") && (maxInput.value != "")) {
-//         errorAlert.removeAttribute('hidden');
```

Comment:
Not sure what this code was. Doesn't look like it relates to this project.

Reply:
Also not sure what that is. I assume deleting it is fine? Maybe just notes someone typed in and forgot to delete?

Reply:
Thats probably it. I'll leave it till tomorrow in case Jeremiah was using it.